.eslintrc

```diff
@@ -688,4 +688,4 @@ rules:
   no-bitwise: 0
 
   # Disallows use of unary operators, ++ and --.
-  no-plusplus: 0
+  no-plusplus: 0
```

Comment:
[Keep the trailing newline](http://stackoverflow.com/questions/729692/why-should-files-end-with-a-newline)